src/pr.tpl

```diff
@@ -3,7 +3,9 @@
 This **draft** PR is opened by Github action [optic-release-automation-action](https://github.com/nearform/optic-release-automation-action).
 
 A new **draft** GitHub release [${releaseMeta.version}](${draftRelease.html_url}) has been created.
-
+<% if (releaseMeta.monorepoPackage) { %>
```

Comment:
instead of just adding this, I think we should consider rewording the existing text of this template instead. If you think it's too early to make this change, ok, but let's track this as an issue.

Reply:
I have created [this issue](https://github.com/nearform/optic-release-automation-action/issues/192) to track the template improvements